lib/app/components/pages/create_chat_group/controllers/create_chat_group_controller.dart

```diff
@@ -1,19 +1,78 @@
 import 'package:favoritism_communication/app/services/services.dart';
+import 'package:favoritism_communication/app/components/organisms/organisms.dart';
+import 'package:favoritism_communication/app/dummy_data/chat_dummy_data.dart';
 import 'package:get/get.dart';
 
 class CreateChatGroupController extends GetxController {
-  RxList searchResult = [].obs;
-  final RxBool isMemberSelected = false.obs;
+  final RxList searchResult = [].obs;
+
+  final isFocusedOnSearchBar = false.obs;
+  final RxList<FriendCardData> friendCardDataList = <FriendCardData>[].obs;
+  final RxList<FriendCardData> selectedMemberDataList = <FriendCardData>[].obs;
+
   final SearchService searchService = Get.find();
 
   @override
   void onInit() {
     searchService.getTargetUsers();
+    fetchFriendData();
+    initSelectedFlag();
     super.onInit();
```

Comment:
```suggestion
    super.onInit();
    debounce(
      searchService.target,
      (_) {
        searchService.prefixSearch(searchService.target.value);
        searchResult.assignAll(searchService.results);
      },
      time: const Duration(seconds: 1),
    );
```
リファクタ③`debounce()`というコールバック関数がGetXに用意されてるのでこちらを追加します

Reply:
@king-kazu39 
ご指摘の通り修正いたしました。